app/javascript/packs/ping_thing/ping_thing_table.vue

```diff
@@ -99,6 +143,19 @@
         } else {
           return " - "
         }
+      },
+      get_filtered_records() {
+        var ctx = this
+
+        axios.get(ctx.api_url + "?time_filter=" + this.filter_time)
```

Comment:
Please add additional params using the object:
```suggestion
        axios.get(
          ctx.api_url,
          { params: { time_filter: this.filter_time }}
        ).then(...)
```